integration-test/src/main/java/com/sequenceiq/it/CustomHTMLReporter.java

```diff
@@ -0,0 +1,52 @@
+package com.sequenceiq.it;
+
+import java.io.File;
+import java.util.List;
+
+import org.apache.velocity.VelocityContext;
+import org.testng.ISuite;
+import org.testng.xml.XmlSuite;
+import org.uncommons.reportng.HTMLReporter;
+import org.uncommons.reportng.ReportNGException;
+
+public class CustomHTMLReporter extends HTMLReporter {
+
+    private static final String INDEX_FILE = "custom.index.html";
+
+    private static final String SUITES_KEY = "suites";
+
+    private static final String ONLY_FAILURES_KEY = "onlyReportFailures";
+
+    private static final String ONLY_FAILURES_PROPERTY = "org.uncommons.reportng.failures-only";
+
+    private static final String SUITES_FILE = "custom.suites.html";
+
+    private static final String REPORT_DIRECTORY = "html";
+
+    @Override
+    public void generateReport(List<XmlSuite> xmlSuites, List<ISuite> suites, String outputDirectoryName) {
+        try {
+            super.generateReport(xmlSuites, suites, outputDirectoryName);
+            boolean onlyFailures = System.getProperty(ONLY_FAILURES_PROPERTY, "false").equals("true");
```

Comment:
I know it comes from the original method but would prefer to change the comparison order (dodging NPEs) and maybe hardcoded strings are unnecessary.

suggestion:
`Boolean.TRUE.toString().equals(System.getProperty(ONLY_FAILURES_PROPERTY, Boolean.FALSE.toString()))`